README.rst

```diff
@@ -0,0 +1,203 @@
+Required packages
+-----------------
+
+To run the PyPI software, you need Python 2.7+ and PostgreSQL
+
+
+Quick development setup
+-----------------------
+
+Make sure you read http://wiki.python.org/moin/CheeseShopDev#DevelopmentEnvironmentHints
+and you have a working PostgreSQL DB.
```

Comment:
Maybe give people a minimal test case to establish whether they do or do not have a working PostgreSQL DB?